blogposts/2021/show-us-your-calendar.md

```diff
@@ -0,0 +1,166 @@
+---
+title: 'Async, remote, and flexible: How 7 engineers rethought their work calendars'
+description: 'Engineers are leading the way toward rethinking the 40 hour work week. Here, seven Sourcegraph team members, all current or former software engineers, share how and why they transformed their work calendars to support their preferred lifestyles.'
```

Comment:
```suggestion
description: 'Engineers are leading the way toward rethinking the 40-hour work week. Here, seven Sourcegraph team members, all current or former software engineers, share how and why they transformed their work calendars to support their preferred lifestyles.'
```

@nickmyyz A last thought: consider mentioning a couple of the work styles here (nonlinear/four-day week) to give people a teaser of the work week structures you cover in the post